src/applications/mhv/secure-messaging/reducers/preferences.js

```diff
@@ -0,0 +1,22 @@
+import { Actions } from '../util/actionTypes';
+
+const initialState = {
+  /**
+   * The list of messages being displayed in the folder view/inbox
+   * @type {array}
+   */
+  signature: undefined,
+};
+
+export const preferencesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case Actions.Preferences.GET_USER_SIGNATURE:
+      return {
+        ...state,
+        signature: action.payload,
+      };
+    case 'b':
```

Comment:
Exactly. Linter would complain and would suggest using `if..else` since there is only one condition. However, this preference reducer will grow once we start pulling that work in the next PI, so keeping this here for consistency.